.github/workflows/main.yaml

```diff
@@ -0,0 +1,15 @@
+name: main
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.22' ]
+    name: go ${{ matrix.go }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+      - run: go list -m | xargs go test -race -v
```

Comment:
I believe `go test -race ./...` should be sufficient.

Reply:
Yes, you're probably right. The only reason for this is when there are multiple packages in the module, but I imagine it won't be the case for this lib.

Simplified. PTAL.